x/paramfilter/test/gov_params_test.go

```diff
@@ -0,0 +1,604 @@
+				want := uint64(2)
+				assert.Equal(want, got)
+			},
+		},
+		{
+			"blob.GasPerBlobByte",
+			testProposal(proposal.ParamChange{
+				Subspace: blobtypes.ModuleName,
+				Key:      string(blobtypes.KeyGasPerBlobByte),
+				Value:    `2`,
+			}),
+			func() {
+				got := suite.app.BlobKeeper.GetParams(suite.ctx).GasPerBlobByte
+				want := uint32(2)
+				assert.Equal(want, got)
+			},
+		},
+		{
+			"blob.GovMaxSquareSize",
+			testProposal(proposal.ParamChange{
+				Subspace: blobtypes.ModuleName,
+				Key:      string(blobtypes.KeyGovMaxSquareSize),
+				Value:    `"2"`,
+			}),
+			func() {
+				got := suite.app.BlobKeeper.GetParams(suite.ctx).GovMaxSquareSize
+				want := uint64(2)
+				assert.Equal(want, got)
+			},
+		},
+		{
+			"blobstream.DataCommitmentWindow",
+			testProposal(proposal.ParamChange{
+				Subspace: bsmoduletypes.ModuleName,
+				Key:      string(bsmoduletypes.ParamsStoreKeyDataCommitmentWindow),
+				Value:    `"100"`,
+			}),
+			func() {
+				got := suite.app.BlobstreamKeeper.GetParams(suite.ctx).DataCommitmentWindow
+				want := uint64(100)
+				assert.Equal(want, got)
+			},
+		},
+		{
+			"consensus.block",
+			testProposal(proposal.ParamChange{
+				Subspace: baseapp.Paramspace,
+				Key:      string(baseapp.ParamStoreKeyBlockParams),
+				Value:    `{"max_bytes": "1", "max_gas": "2"}`,
+			}),
+			func() {
+				gotMaxBytes := suite.app.BaseApp.GetConsensusParams(suite.ctx).Block.MaxBytes
+				wantMaxBytes := int64(1)
+				assert.Equal(wantMaxBytes, gotMaxBytes)
+
+				gotMaxGas := suite.app.BaseApp.GetConsensusParams(suite.ctx).Block.MaxGas
+				wantMaxGas := int64(2)
+				assert.Equal(wantMaxGas, gotMaxGas)
+			},
+		},
+		{
+			"consensus.evidence",
+			testProposal(proposal.ParamChange{
+				Subspace: baseapp.Paramspace,
+				Key:      string(baseapp.ParamStoreKeyEvidenceParams),
+				Value:    `{"max_age_duration": "1", "max_age_num_blocks": "2", "max_bytes": "3"}`,
+			}),
+			func() {
+				gotMaxAgeDuration := suite.app.BaseApp.GetConsensusParams(suite.ctx).Evidence.MaxAgeDuration
+				wantMaxAgeDuration := time.Duration(1)
+				assert.Equal(wantMaxAgeDuration, gotMaxAgeDuration)
+
+				gotMaxAgeNumBlocks := suite.app.BaseApp.GetConsensusParams(suite.ctx).Evidence.MaxAgeNumBlocks
+				wantMaxAgeNumBlocks := int64(2)
+				assert.Equal(wantMaxAgeNumBlocks, gotMaxAgeNumBlocks)
+
+				gotMaxBytes := suite.app.BaseApp.GetConsensusParams(suite.ctx).Evidence.MaxBytes
+				wantMaxBytes := int64(3)
+				assert.Equal(wantMaxBytes, gotMaxBytes)
+			},
+		},
+		{
+			"consensus.Version.AppVersion",
+			testProposal(proposal.ParamChange{
+				Subspace: baseapp.Paramspace,
+				Key:      string(baseapp.ParamStoreKeyVersionParams),
+				Value:    `{"app_version": "3"}`,
+			}),
+			func() {
+				got := *suite.app.BaseApp.GetConsensusParams(suite.ctx).Version
+				want := tmproto.VersionParams{AppVersion: 3}
+				assert.Equal(want, got)
+			},
+		},
```

Comment:
interesting. In theory we should want this to not be changed through governance, since this is coordinated by the upgrade protocol

Reply:
Agreeeeeed. IMO most of the params that are currently gov modifiable we should make unmodifiable. If they remain gov modifiable then we could enter a scenario where off chain rough consensus agrees on a change that diverges from how on-chain token voting governance votes.